crates/brainfuck_prover/src/brainfuck_air/mod.rs

```diff
@@ -112,22 +124,27 @@ pub fn prove_brainfuck() -> Result<BrainfuckProof<Blake2sMerkleHasher>, ProvingE
     let channel = &mut Blake2sChannel::default();
     let commitment_scheme =
         &mut CommitmentSchemeProver::<_, Blake2sMerkleChannel>::new(config, &twiddles);
-    let tree_builder = commitment_scheme.tree_builder();
+    let mut tree_builder = commitment_scheme.tree_builder();
 
-    // ┌──────────────────────────┐
-    // │   Interaction Phase 0    │
-    // └──────────────────────────┘
+    // ┌───────────────────────────────────────┐
+    // │    Interaction Phase 0 - Main Trace   │
+    // └───────────────────────────────────────┘
+    let vm_trace = inputs.get_trace();
```

Comment:
In rust this is common to not put get in getter, like `get_trace` becoming `trace` but this is totally fine with me to keep as is, just a note for the future maybe :)

Reply:
Okay, good to know!
I can refactor it in an small upcoming PR (with the other getter that might use `get_`

Reply:
for sure :)